sdk/eventhub/azure-eventhub/azure/eventhub/_consumer_client.py

```diff
@@ -102,7 +102,10 @@ class EventHubConsumerClient(ClientBase):
      evaluation regardless of the load balancing strategy.
      Greedy strategy is used by default.
     :paramtype load_balancing_strategy: str or ~azure.eventhub.LoadBalancingStrategy
-
+    :keyword str custom_endpoint_address: The custom endpoint address to connect to. Default is None.
```

Comment:
I think this description could probably be more descriptive about the specific scenario this supports. I don't want people to fall into it by accident :)

Reply:
description updated